crates/noirc_abi/src/input_parser/mod.rs

```diff
@@ -10,6 +10,7 @@ use crate::{Abi, AbiType};
 /// This is what all formats eventually transform into
 /// For example, a toml file will parse into TomlTypes
 /// and those TomlTypes will be mapped to Value
+#[cfg_attr(test, derive(PartialEq))]
```

Comment:
I've put this behind a flag as we only use it during tests. It's not an unhelpful thing to have so we could just always derive this.